docs/cli.md

```diff
@@ -683,6 +688,7 @@ viam module upload --version=1.0.0 --platform=darwin/arm64 packaged-module.tar.g
 <!-- prettier-ignore -->
 | Command option | Description | Positional arguments |
 | -------------- | ----------- | -------------------- |
+| `generate` | Auto-generate stub files for a new Python module by following prompts. | - |
```

Comment:
This will have Go soon, no need to specifiy Python here
```suggestion
| `generate` | Auto-generate stub files for a new module by following prompts. | - |
```

Reply:
In fact golang is already there and supported

Reply:
Hooray!